cloudinit/sources/DataSourceAzure.py

```diff
@@ -1836,9 +1836,7 @@ def read_azure_ovf(contents):
     if ovf_env.password:
         defuser["lock_passwd"] = False
         if DEF_PASSWD_REDACTION != ovf_env.password:
-            defuser["passwd"] = cfg["password"] = encrypt_pass(
-                ovf_env.password
-            )
+            defuser["passwd"] = encrypt_pass(ovf_env.password)
```

Comment:
@TheRealFalcon so is this really just the suggestion then below?
```suggestion
            defuser["hashed_passwd"] = encrypt_pass(ovf_env.password)
```